.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring
+# files. If you find yourself ignoring temporary files generated by your text
+# editor or operating system, you probably want to add a global ignore instead.
+# For example:
+#
+#     git config --global core.excludesfile ~/.gitignore_global
+#     echo *.swp >> ~/.gitignore_global
+#
+
+# Compiled python code.
+*.pyc
+
+# Upgrade log files
+upgrade_highlights
+full_upgrade
+
```

Comment:
Please add "preupgrade_cli", "preupgrade_api", "postupgrade_cli", "postupgrade_api"  in ignore list.